### PR TITLE
fix: Improve util functions type

### PR DESCRIPTION
**Pull request by HoonBaek**

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -415,18 +415,26 @@ export const hasSameMembers = <T>(a: T[], b: T[]): boolean => {
...
 
 export const filterMessageListParams = (
   params: MessageListParams,
   message: SendableMessageType,
 ): boolean => {
+  const {
+    customTypesFilter = [],
+  } = params as MessageListParams & {
+    customTypesFilter: Array<string>;
+  };
```
Comment (HoonBaek) [inline src/utils/index.ts:431]: Does this look okay?
@bang9

Comment (bang9) [reply src/utils/index.ts:431]: I believe we don't need `as MessageListParams & {
    customTypesFilter: Array<string>;
  };`

Comment (HoonBaek) [reply src/utils/index.ts:431]: Oh, that's true, thanks!

**Review by AhyoungRyu — APPROVED** [review]

File: src/utils/index.ts [hunk 2]
```diff
@@ -320,7 +320,7 @@ export const isMessageSentByMe = (
   userId: string,
   message: SendableMessageType,
 ): boolean => (
-  (userId && message?.sender?.userId) && userId === message.sender.userId
+  !!(userId && message?.sender?.userId && userId === message.sender.userId)
```
Comment (liamcho) [inline src/utils/index.ts:323]: why do we need to do this?

**Pull request merged**